source/npm/util.js

```diff
@@ -109,3 +111,57 @@ exports.checkIgnoreStrategy = ({files}) => {
 		`);
 	}
 };
+
+// New files added part of .npmignore or not part of files-attribute in "package.json"
+exports.checkNewFiles = async (newFiles, filesFromFileAttribute) => {
+	const rootDir = pkgDir.sync();
+	const npmignoreExists = fs.existsSync(path.resolve(rootDir, '.npmignore'));
```

Comment:
Let's create a separate method for this, as the exact same code is used twice (both here and in [`checkIgnoreStrategy`](https://github.com/sindresorhus/np/pull/456/files#diff-5262e3d1f54171794381aaf21e3dac4eR104-R113).

Reply:
Separate function is created